ckanext/benap/i18n/fr/LC_MESSAGES/ckanext-benap.po

```diff
@@ -33,7 +33,15 @@ msgstr "La date de début doit être antérieure à la date de fin"
 
 #: ckanext/benap/validators.py:75
 msgid "URL {url} has to start with https://"
-msgstr ""
+msgstr "L'URL {url} doit débuter par https://"
+
+#: ckanext/benap/validators.py:92
+msgid "Modification must done by a system administrator"
```

Comment:
Misschien beter: "Modification must be done by a system administrator." (dus ook met een puntje erachter)